### DISCUSSION TITLE
Privacy information about GitHub desktop

### DISCUSSION THREAD

**Discussion by Daniel900** [community/community#113318] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Does Github Desktop also transmit confidential Code to GitHub, if I use it in combination with our own gitlab repository?

### Body

Hey,

I wanted to use the Github Desktop Application in my Company, but I didn't find a privacy statement specific for it. Unfortunately the official privacy statement talks about transmitting code to GitHub.

Does Github Desktop also transmit confidential Code to GitHub, if I use it in combination with our own gitlab repository?

Best regards

**Comment by nickkipshidze** [comment]
Yes you can make changes to a GitLab repository using GitHub desktop. Check this out: https://learn.hibbittsdesign.org/gitlab-githubdesktop/cloning-a-gitlab-repo
